bundles/org.openhab.binding.sonyprojector/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="sonyprojector" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+	<name>SonyProjector Binding</name>
+	<description>The SonyProjector binding controls a Sony projector through Ethernet ()PJ Talk) or serial.</description>
+	<author>Markus Wehrle &amp; Laurent Garnier</author>
```

Comment:
We usually leave this element out if there is more than one author - anyhow, we will soon remove the author tag from all bindings, so you might want to not add it at all in the first place.

Reply:
I hope it does not crash in Paper UI if no author is filled. I suppress it.